app/models/dream.rb

```diff
@@ -33,4 +40,20 @@ def keywords
     	puts 'Error in keyword extraction call: ' + response['statusInfo']
     end
   end
+
+  def keyword_analysis
+    alchemyapi = AlchemyAPI.new()
+    response = alchemyapi.keywords('text', text, { 'sentiment'=>1 })
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator '=>'.